File: 01TwoSum/TwoSum.swift
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var dict: [Int: Int] = [Int: Int]()
        for i in 0..<nums.count {
            if let complementValue = dict[target - nums[i]] {
                return [complementValue, i]
            }
            dict[nums[i]] = i
        }
        return [-1]
    }

File: 01TwoSum/TwoSum.py
from typing import List


def two_sum(self, nums: List[int], target: int) -> List[int]:
    hash_map = {}
    for idx, num in enumerate(nums):
        if target - num in hash_map:
            return [hash_map[target - num], idx]
        hash_map[num] = idx
    return [-1]
